hotham/src/shaders/pbr.glsl

```diff
@@ -56,22 +56,25 @@ vec3 tonemap(vec3 color) {
 
 // Get normal, tangent and bitangent vectors.
 vec3 getNormal(uint normalTextureID) {
-    vec3 n, t, b, ng;
-
-    // Trivial TBN computation, present as vertex attribute.
-    // Normalize eigenvectors as matrix is linearly interpolated.
-    t = normalize(inTBN[0]);
-    b = normalize(inTBN[1]);
-    ng = normalize(inTBN[2]);
-
-    if (normalTextureID != NOT_PRESENT) {
-        vec3 ntex;
-        ntex.xy = texture(textures[normalTextureID], inUV).ga * 2.0 - 1.0;
-        ntex.z = sqrt(1 - dot(ntex.xy, ntex.xy));
-        return normalize(mat3(t, b, ng) * ntex);
-    } else {
-        return ng;
+    vec3 N = normalize(inNormal);
+    if (normalTextureID == NOT_PRESENT) {
+        return N;
     }
+
+    vec3 textureNormal;
```

Comment:
Could you add a couple comments here? Just something like "compute the tangent based on..". May also be worth noting the performance benefit in case someone looks at this and thinks it may be slower